tests/virtualization/xen/open_vm_tools.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Simple vmware client testing with updated open-vm-tools
+# Maintainer: Michael Grifalconi <mgrifalconi@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils;
+use xen;
+
+sub run {
+    my ($self) = @_;
+
+    assert_script_run "ssh root\@$_ zypper -n in open-vm-tools" foreach (keys %xen::guests);
```

Comment:
Sometimes problems with `zypper in` in virtualization runs occur due to various issues (guest not yet booted, network hickups, ecc.). Here `script_retry` has proven to be more robust - I suggest to change this to

    script_retry "ssh root\@$_ 'zypper -n open-vm-tools'", delay => 30, retry => 6 foreach (keys %xen::guests);